.github/workflows/build_test_deploy.yml

```diff
@@ -514,7 +514,7 @@ jobs:
       - run: echo "CNA_CHANGE<<EOF" >> $GITHUB_OUTPUT; echo "$(node scripts/run-for-change.js --type cna --always-canary --exec echo 'yup')" >> $GITHUB_OUTPUT; echo "EOF" >> $GITHUB_OUTPUT
         id: cna-change
 
-      - run: docker run --rm -v $(pwd):/work mcr.microsoft.com/playwright:v1.28.1-focal /bin/bash -c "cd /work && ls && curl https://install-node.vercel.app/v16 | FORCE=1 bash && node -v && npm i -g pnpm@${PNPM_VERSION} && NEXT_TEST_JOB=1 NEXT_TEST_CNA=1 xvfb-run node run-tests.js test/integration/create-next-app/index.test.ts test/integration/create-next-app/templates.test.ts >> /proc/1/fd/1"
+      - run: docker run --rm -v $(pwd):/work mcr.microsoft.com/playwright:v1.28.1-focal /bin/bash -c "cd /work && curl -s https://install-node.vercel.app/v16 | FORCE=1 bash && npm i -g pnpm@${PNPM_VERSION} > /dev/null && NEXT_TEST_JOB=1 NEXT_TEST_CNA=1 xvfb-run node run-tests.js test/integration/create-next-app/index.test.ts test/integration/create-next-app/templates.test.ts >> /proc/1/fd/1"
```

Comment:
Why does this one have `> /dev/null` but the others don't?

Also, is that going to hide the error message when npm registry is down?

Reply:
Because other commands have silent flags, like `-s` for curl.
`> /dev/null` redirects just stdout, I expect there to be printed to stderr.
Blt haven't checked.

Reply:
Because other commands have silent flags, like `-s` for curl.
`> /dev/null` redirects just stdout, I expect there to be printed to stderr.
Blt haven't checked.

Reply:
Yup, errors go into stderr